package.json

```diff
@@ -5,7 +5,9 @@
   "main": "app.js",
   "scripts": {
     "start": "nodemon app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint .",
+    "format": "eslint . --fix"
```

Comment:
Adds `npm run format` option. This can be used to add easily format your files*.

* Won't fix all errors.